arch/arm/src/stm32f7/stm32_can.c

```diff
@@ -322,7 +323,7 @@ static uint32_t stm32can_vgetreg(uint32_t addr)
         {
           /* Yes.. then show how many times the value repeated */
 
-          caninfo("[repeats %d more times]\n", count - 3);
+          caninfo("[repeats %" PRId32 " more times]\n", count - 3);
```

Comment:
```suggestion
          caninfo("[repeats %" PRIu32 " more times]\n", count - 3);
```
`count` has `uint32_t` type

Reply:
@gustavonihei - I was torn on matching the old code usage and fixing them. 

Are you you are siding on the "fixing them" side?

Reply:
Since it is already being changed, I believe it would be best if changed to the correct one :)

Creating another PR just for changing it again seems a waste of resources (review time, CI usage...)